engines/instance_verification/lib/instance_verification/engine.rb

```diff
@@ -1,16 +1,18 @@
 require 'fileutils'
 
 module InstanceVerification
-  def self.update_cache(remote_ip, system_login, product_id, _is_byos)
+  def self.update_cache(remote_ip, system_login, product_id, is_byos: false, registry: false) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
Can't we just drop the unused `is_byos` argument. I am doubtful the solution to the reduce the number of requests to scc with the same set of credentials will include a simple flag.

Reply:
Yes, I will take care of removing that in a different PR

>  I am doubtful the solution to the reduce the number of requests to scc with the same set of credentials will include a simple flag

Well, not the place to talk about it but the solution could follow the same approach as repository|registry and 
has its own directory
`REPO_PAYG` and REPO_BYOS` with its own timer for the scrubber, in any case, that flag will be removed in a different PR

Reply:
I would be surprised if the solution looks like that.

Reply:
Still would like to see the rubocop exception dropped, as the need for `is_byos` as an argument is speculation. ANyway, given the time constraints, this is more os a nitpick, thus approving this for merge.